components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/DefaultsCustomizer.java

```diff
@@ -49,7 +51,8 @@
   @Inject
   public DefaultsCustomizer(
       final UserAgentGenerator userAgentGenerator,
-      @Named("${nexus.httpclient.requestTimeout:-30s}") final Time requestTimeout,
+      @Named("${nexus.httpclient.requestTimeout:-20s}") final Time requestTimeout,
+      @Named("${nexus.httpclient.connectionRequestTimeout:-30s}") final Time connectionRequestTimeout,
```

Comment:
The property name is matching the builder name so I suggest we keep the same name to avoid confusion.